app/routes/details/index.ts

```diff
@@ -24,7 +24,10 @@ module.exports = [
       },
       handler: async (request: Request, h: ResponseToolkit): Promise<ResponseObject> => {
         request.query.searchString=encodeURIComponent(request.query.searchString)
-        return h.view('details/index', await detailsModel(request.query as queryParams))
+        return h.view('details/index', {
+          ...(await detailsModel(request.query as queryParams)),
+          relatedContentData: getRelatedContentLinks('details'),
```

Comment:
Since we already have a module which puts together a model for details page (detailsModel), it would make sense to put code related to relatedContentData in there.

Reply:
Thanks FYC, Done!